package.json

```diff
@@ -41,7 +41,9 @@
     "ownerslint": "ts-node --project scripts/ scripts/ownerslint.ts",
     "tslint": "tslint -c tslint.json --project ./tsconfig.json",
     "stylelint": "stylelint \"src/**/*.+(css|scss)\" --config .stylelintrc.json --syntax scss",
-    "resync-caretaker-app": "ts-node --project scripts scripts/caretaking/resync-caretaker-app-prs.ts"
+    "resync-caretaker-app": "ts-node --project scripts scripts/caretaking/resync-caretaker-app-prs.ts",
+    "ts-circular-deps:check": "yarn -s ts-circular-deps check ./goldens/ts-circular-deps.json --base-dir ./src/ --glob \"src/**/*.ts\"",
+    "ts-circular-deps:approve": "yarn -s ts-circular-deps approve ./goldens/ts-circular-deps.json --base-dir ./src/ --glob \"src/**/*.ts\""
```

Comment:
I'll make a note. Maybe we can pass these options always and share a script that sets the `--glob`.

Reply:
Yeah, that works too. I'm not sure what our policy should be. I could imagine us both supporting the CLI, and the config. CLI flags make testing and consumption of these tools easy.

Reply:
I agree that the CLI has a lot of value.

I think I am envisioning a place where in all of our repos, the default configurations are done via the config file, so you can literally just run the exact same commands in any of the repos and don't have to know anything extra.

Reply:
Yeah, this makes sense to me. Maybe a balance of both is what we want here. I don't think we only want config support as it makes testing etc. difficult. Also I like that tools are self-contained and could be moved anywhere without being coupled to a Angular dev-infra specific config.

It would be cool if our tools remain self-contained, provide their own entry-point, but just the dev-infra CLI delegates accordingly based on the config? Sounds like something we can discuss in the next meeting / or per chat.

Reply:
Especially for tools like that it's also common that more goldens exist. In those cases, I'd personally always prefer the setup next to where the tool is launched.